lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -2113,4 +2143,34 @@ static SubscriberQuotaUpdate make_subscriber_quota_update(
   update.set_update_type(state);
   return update;
 }
+
```

Comment:
I think that we should find a better way to identify who this rule belongs too... Right now we are using ip for that. In case of cwag we don't receive an IP, but if we store the IP in the future, that will break this functions (that is why for cwag case I used a different condition)

Maybe with the introduction of tunnelid it will be more homogeneous for both lte and cwag case

Reply:
Yes I agree tunnelID would be better